### PR TITLE
Feat/search preview

### PR DESCRIPTION
**Pull request by ImanRezasefat**

### CODE REVIEW

File: src/Nowruz/modules/layout/index.tsx [hunk 1]
```diff
@@ -24,6 +25,7 @@ export const Layout = () => {
         <div className="w-full fixed top-0 right-0 z-20 bg-Base-White ">
           <HeaderNavBar setOpen={setOpen} logout={logOut} />
         </div>
+        <SearchModal open={true} onClose={() => console.log} />
```
Comment (jeyem) [inline src/Nowruz/modules/layout/index.tsx:28]: on close should accept undefined and do nothing for production use log is bad practice

File: src/Nowruz/pages/test/index.tsx
Comment (marjan-ara) [inline src/Nowruz/pages/test/index.tsx:1]: what is this page?

File: src/Nowruz/modules/layout/index.tsx (hunk 1 above)
Comment (ImanRezasefat) [reply src/Nowruz/modules/layout/index.tsx:28]: This is used for testing purposes for azin and shall be removed after test is passed.

File: src/Nowruz/modules/Search/components/ResultList/index.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+import css from './result-list.module.scss';
+import { ResultListProps } from './ResultList.types';
+export const ResultList: React.FC<ResultListProps> = ({ list, onSelect }) => {
+  const selectedRef = useRef(null);
+  const [selectedRowIndex, setSelectedRowIndex] = useState(null);
+  const [hoveredRowIndex, setHoveredRowIndex] = useState<null | number>(null);
+  const navigate = useNavigate();
+
+  //Comments are footbar feature for list may be back later
```
[the remark this answers is not in the document]
Comment (ImanRezasefat) [reply src/Nowruz/modules/Search/components/ResultList/index.tsx:13]: As mentioned in the comment this feature can be used in the future.

File: src/Nowruz/pages/test/index.tsx
Comment (ImanRezasefat) [reply src/Nowruz/pages/test/index.tsx:1]: You added this page to test nowruz layout :)

Comment (marjan-ara) [reply src/Nowruz/pages/test/index.tsx:1]: Sorry, I'll delete it in the next commit

Comment (ImanRezasefat) [reply src/Nowruz/pages/test/index.tsx:1]: let's keep it as a playground for testing purposes it is not linked to/from anywhere

**Review by marjan-ara — APPROVED** [review]

**Pull request merged**